README.md

```diff
@@ -1 +1,83 @@
-# OOP-mini-projects-set-up-associations
\ No newline at end of file
+# school_library_associations_training
+
+## About
+
+This is a code review training project for Ruby/Rails.
+
+There are 7 classes and these classes are connected through database associations.
+
+## Built with
+* Postgresql
+* rubocop linters
+- <code><img height="20" src="https://www.ruby-lang.org/images/header-ruby-logo.png"></code><br>
+-  <code><img height="20" src="https://rubyonrails.org/images/rails-logo.svg"></code><br>
+- ![Git](https://img.shields.io/badge/-Git-05122A?style=flat&logo=git)&nbsp;<br>
+- ![GitHub](https://img.shields.io/badge/-GitHub-05122A?style=flat&logo=github)&nbsp;<br>
+- ![Visual Studio Code](https://img.shields.io/badge/-Visual%20Studio%20Code-05122A?style=flat&logo=visual-studio-code&logoColor=007ACC)&nbsp;
+## Getting Started
+
+## Have a copy
+
+## Getting started
+
+To get started, you should first get this file in your local machine by cloning or forking this project or typing in your terminal
+```
+ git clone https://github.com/elisha2kyakpo1/school_library_associations_training.git
+ cd school_library_associations_training
+```
+Before you start using the app, you need to be sure that you have Ruby installed on your computer, by typing.
+```
+ ruby -v
+```
+You should see your version of ruby.
+If it's not installed in your system, follow this [guide](https://www.ruby-lang.org/en/documentation/installation/) and it will help you to get it done.
+
+Then you can install rails using this command,
+```
+ gem install rails
+```
+And you can also check your rails version by typing in your terminal
+```
+ rails -version
+```
+After that do bundle install to install all required gems and dependences
+```
+ bundle install
+```
+Install yarn by running the command below
+```
+ yarn install
+```
+Since we are using postgresql database, run this command in your tenimal to create the database
+```
+ rails db:create
+```
+Finally, run the migration to populate your database to have the tables
+```
+ rails db:migrate
+```
+You then interract with the program from rails server on the web by
+```
+ rails s
+```
+Open your favourite blowser and type the command below
+```
+ http://localhost:3000
+```
+## Contributors
+
+## Elisha Kyakopo
+
+  Linkedin: [Elisha Kyakopo](https://www.linkedin.com/in/elisha-kyakopo/)
+
+  Github: [@elisha2kyakpo1](https://github.com/elisha2kyakpo1)
+
+  Twitter: [@elisha1k](https://twitter.com/Elisha1k)
+
+## Contributions
+
+Contributions, issues and requests are welcome
```

Comment:
- [ ] <b>OPTIONAL:</b> It would be nice to add a link to your issues page so that it would be easy for anyone to find your issues page.